articles/getting_started/1_setting_up_your_os_for_development_macos.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting up your OS for development on MacOS
+description: This section provides a step-by-step guide for setting up your development environment on Mac.
+---
+
+To develop with MonoGame in C#, you will need to install the .NET SDK. As of MonoGame 3.8.2 the minimum supported version is .NET 8.
+
+## Install .NET 8 SDK
+
+1. Navigate to [https://dotnet.microsoft.com/en-us/download/dotnet/8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
+2. Download the `.NET SDK` Installer for your current platform of `macOS`
+3. Once the **.pkg** file finishes downloading, run it and follow the prompts to install the .NET SDK
+
+    ![Download .NET For Windows](./images/1_setting_up_your_development_environment/dotnet_8_download_page.png)
+
+# Apple Silicon Known Issues
+
+ For the time being, MonoGame requires that you install the x64 version of the .NET runtime if you are running on an Apple Silicon mac in order to be able to build content. It is also required that [Rosetta](https://support.apple.com/en-us/HT211861) is enabled. 
+
+ 1. Navigate to [https://dotnet.microsoft.com/en-us/download/dotnet/8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
+ 2. Download the .NET Runtime Installer for macOS x64
+ 3. Once the **.pkg** file finishes downloading, run it and follow the prompts to install the .NET Runtime
```

Comment:
There are other steps needed for this  see the old site https://docs.monogame.net/articles/getting_started/1_setting_up_your_development_environment_vscode.html?tabs=windows#apple-silicon-known-issues. Not sure where the best place to put them are though?

Reply:
Most of what was in that original article @dellis1972 has been moved to specific docs, like the "Setting up mac" doc